catalog/service/common/src/main/java/org/projectnessie/catalog/service/config/CatalogConfig.java

```diff
@@ -17,12 +17,23 @@
 
 import static org.projectnessie.catalog.service.config.CatalogConfig.removeTrailingSlash;
 
+import java.net.URI;
 import java.util.Map;
 import java.util.Optional;
 import org.projectnessie.nessie.docgen.annotations.ConfigDocs.ConfigPropertyName;
 
 public interface CatalogConfig {
 
+  /**
+   * Optional override for the Catalog Server's base URI. If not set, URI info from the application
+   * server will be used.
```

Comment:
```suggestion
   * The base URI used by Iceberg REST clients to contact Nessie, needs to be specified if Nessie runs behind a proxy and/or load-balancer.
```

I guess, an example would be nice here.

Reply:
added

Reply:
I did not rephrase the javadoc because the override is not strictly necessary. It is needed only if the proxy changes port or URI path.